docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enabled. You can check it with `Settings.canDrawOverlays(this);`. This is required because, if your app produces an error in the react native component, the error view is displayed above all the other windows. Due to the new permissions system, introduced in the api level 23, the user needs to approve it.
```

Comment:
I am mobile developer and I know that I need to add permissions to access location or even the network state in Android. I also know that in iOS to access user location I need to include the UIRequiredDeviceCapabilities key in the app’s Info.plist file. That is in the docs.

I am starting at React Native, so this problem is probably mine, but, I didn't know that to an `Hello World` app integration (and probably the other Android Developers or React Native/JS developers also) I need to request the overlay permission. And since the app just crashes and the crash report in the `logcat` is not the most helpful, IMO, this information would be very helpful to other developers.

Most of the Android developers will not use an emulator, so when they open the `ReactActivity` they will just see a crash because the activity is not connected with the NPM server. They need to open the React Native activity the first time to access to the dev activity to configure the server url.

About the run time permissions, introduced in Android API level 23, there is a section in the documentation ([PermissionsAndroid](http://facebook.github.io/react-native/docs/permissionsandroid.html)) so, the developers know that they need to handle it. My point is: it is not easy to get, that if you are targeting to Android API level 23, you will need to request the overlay permission in development mode, even for an `Hello World`.

Please don't get me wrong. I am just trying to contribute to the community and improve the documentation to help other developers to have a better learning curve when they start in React Native.